src/core_modules/capture-core/components/WorkingLists/TeiWorkingLists/Setup/TeiWorkingListsSetup.component.js

```diff
@@ -136,6 +138,95 @@ const useFiltersOnly = ({
         ];
     }, [enrollmentDateLabel, incidentDateLabel, showIncidentDate, stages]);
 
+
+const useProgramStageFilters = ({ stages }: TrackerProgram, programStageId?: string) => {
+    const supportsProgramStageWorkingLists = useFeature(FEATURES.programStageWorkingList);
+    const programStage = programStageId && stages.get(programStageId);
+    const occurredAtLabel = programStage
+        // eslint-disable-next-line no-underscore-dangle
+        ? programStage.stageForm._labels.occurredAt
```

Comment:
hey @simonadomnisoru I think we can use `stageForm.getLabel('occurredAt')` here instead of calling the private variables. WDYT?